packages/eslint-plugin/src/rules/no-triple-slash-reference.ts

```diff
@@ -11,7 +11,7 @@ export default util.createRule({
     },
     schema: [],
     messages: {
-      tripleSlashReference: 'Do not use a triple slash reference.',
+      noTripleSlashReference: 'Do not use a triple slash reference.',
     },
```

Comment:
add `deprecated: true`, and `replacedBy: ['triple-slash-reference']` to the meta